locale/es.yml

```diff
@@ -1248,113 +1248,113 @@ __TRAIT__: ""
 # |  LANGUAGE: PYTHON  |
 # ----------------------
 # << Keywords >>
-  False: ""
-  None: ""
-  True: ""
-  and: ""
-  as: ""
-  assert: ""
-  async: ""
-  await: ""
-  break: ""
-  class: ""
-  continue: ""
-  def: ""
-  del: ""
-  elif: ""
-  else: ""
-  except: ""
-  finally: ""
-  for: ""
-  from: ""
-  global:  ""
-  if: ""
-  import: ""
-  in: ""
-  is: ""
-  lambda: ""
-  nonlocal: ""
-  not: ""
-  or: ""
-  pass: ""
-  raise: ""
-  return: ""
-  try: ""
-  while: ""
-  with: ""
-  yield: ""
+  False: "Falso" # ✅✅✅✅✅✅✅
+  None: "Ninguno" # ✅✅✅✅✅✅✅
+  True: "Verdadero" # ✅✅✅✅✅✅✅
+  and: "Y" # ✅✅✅✅✅✅✅
+  as: "como" # ✅✅✅✅✅✅
+  assert: "afirmar" # ✅❓ What about "asegurar"? ✅ I agree with "afirmar" ✅ ✅ I also agree with "afirmar"✅✅ I agree with "afirmar"
+  async: "asíncrono" # ✅✅✅✅✅✅✅
+  await: "esperar" # ✅✅✅✅✅✅✅
+  break: "interrumpir" # ✅✅✅✅✅Also "romper"✅ Yes, it also could be "romper"
+  class: "clase" # ✅✅✅✅✅✅✅
+  continue: "continuar" # ✅✅✅✅✅✅✅
+  def: "define" # ✅ def it's a short for define, in the traslated version isn't shorted ❓What about using "define" as imperative, because you're defining something? ✅ I agree with "define" ✅✅ I also agree with "define"✅✅
+  del: "eliminar" # ✅✅ same as def ✅ I agree with "eliminar" ✅ could be left as del, but eliminar is good as well✅✅✅Also "borrar"
+  elif: "si no, si" # ✅❓ I added a comma here, it's something like, if this does not satisfy let's try this other ✅ I agree with the comment, it is better having the comma ✅ I agree that having the comma may be beneficial, but it might also be a bit too long (maybe something like no, si is fine, similar to else if)✅✅✅✅
```

Comment:
The phrase si no, si is the best direct translation - however, it makes the one keyword lengthy. I think no, si or no si is a good way to go to help new users learn the terms a little easier (could be made into nosi like elif is currently defined). Keeping it this way would also be consistent with the translation of if being si.